dissect/target/loaders/log.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+
+from dissect.target import Target
+from dissect.target.filesystem import VirtualFilesystem
+from dissect.target.loader import Loader
+
+
+class LogLoader(Loader):
+    LOGS_DIRS = {
+        "EvtxPlugin": "sysvol/windows/system32/winevt/logs",
+        "EvtPlugin": "sysvol/windows/system32/config",
+    }
+
+    @staticmethod
+    def detect(path: Path) -> bool:
+        return False
+
+    def _map_entry(self, entry: Path) -> str:
```

Comment:
I'm not sure this warrants a separate function. The individual functions are small enough that I feel like it makes it a bit harder to follow right now.